gpii/node_modules/contextManager/src/ContextManager.js

```diff
@@ -169,6 +169,7 @@
 
         lifecycleManager.update(finalPayload, function (response) {
             if (response.success) {
+                delete finalPayload.lifecycleInstructions;
```

Comment:
Remind why we do this delete? Will this affect @simonbates work on the untrusted local flowmanager?